server/auth/auth.controller.js

```diff
@@ -1,10 +1,16 @@
 function handlePassportLogin(req, res) {
+  /* eslint-disable no-param-reassign */
```

Comment:
`/* eslint-disable-next-line no-param-reassign */` if much cleaner and you only need one comment 👍